src/core/models/searchconfig.js

```diff
@@ -9,7 +9,13 @@ export default class SearchConfig {
      * Also defines the number of results per page, if pagination is enabled
      * @type {number}
      */
-    this.limit = config.limit || 20;
+    this.limitForVertical = config.limit || 20;
+
+    /**
+     * The max results per vertical for universal search.
+     * @type {Object<string, number>}
+     */
+    this.universalLimit = config.universalLimit;
```

Comment:
I think these are showing up as changed because we tried the merge instead of force push, if you reset --hard the merge commit (which had no code changes) and just do a force push do these go away?